includes/blocks/class-sensei-page-blocks.php

```diff
@@ -18,6 +18,44 @@ class Sensei_Page_Blocks extends Sensei_Blocks_Initializer {
 	 */
 	public function __construct() {
 		parent::__construct( [ 'page' ] );
+
+		add_filter( 'render_block_core/post-featured-image', [ $this, 'add_a_badge'], 10, 3 );
+		add_filter( 'render_block', array( $this, 'add_course_featured_badge' ), 11, 3 );
+	}
+	public function add_a_badge( string $block_content, array $block, WP_Block $instance ) {
```

Comment:
This function could use PHP comments. We should also make its access `@private`. And could we name it `add_badge` instead?

Reply:
a679a7877e675c77f3b369c7905e4e8249552f81